src/index.css

```diff
@@ -1,13 +1,109 @@
-body {
+@import url(https://fitsu-ui-design.netlify.app/css/components.css);
+
+
+
+
+:root{
+  --primary-color: #fc774a;
+  --primary-hover: #e0582b;
+  --secondary-color: rgb(37, 37, 37);
+  --secondary-hover: rgb(49, 49, 49);
+  --font-primary-color: black;
+  --font--secondary-color: rgb(58, 58, 58);
+  --lg-color: white;
+  --body-background-color: #f8f8f8;
+  --light-gray: rgb(184, 182, 182);
+  --success-color : green;
+  --yellow-color : rgb(255, 180, 0);
```

Comment:
Be consistent in providing colors, Either use hex code or RGB value.

Reply:
I will use RGB value for all at the end.. As I am not liking the color combination I used in UI I will change it a bit and fix this part as well.